specs/pvs/corla.pvs

```diff
@@ -254,6 +271,9 @@ BEGIN
   full_hand_count_contest: TYPE FROM contest
 % @todo Contest name uniqueness.
 
+% @todo kiniry These types will be refined when we review the
+% information provided by CDOS on 20 July 2017 about Dominion's file
+% formats.
   outcome: TYPE
```

Comment:
I think `wrong_outcome` should be derived from `contest_outcome`, not `election_outcome`, which I don't think we know enough to define.

I agree, we should deferring the issue of multiple kinds of winners until we have CVR data that we know can express it.